acceptance/tests/mesh_v2/mesh_inject_test.go

```diff
@@ -91,6 +98,8 @@ func TestMeshInject_MultiportService(t *testing.T) {
 				})
 			}
 
+			// TODO: consider adding a trafficpermission to a particular port and validate
+
 			// Check connection from static-client to multiport.
 			if cfg.EnableTransparentProxy {
```

Comment:
Do we want to check at the end of the test deleting the pod and ensuring the ACL token no longer exists in consul? Totally fine to defer that check to a future PR too.

Reply:
Yeah, I think I'll defer that to another PR, but I will add a TODO as well.